dockstore-client/src/main/java/io/dockstore/client/cli/nested/AbstractEntryClient.java

```diff
@@ -1079,8 +1079,9 @@ WorkflowExecutionServiceApi getWorkflowExecutionServiceApi() {
      * This will attempt to launch a workflow given the command arguments
      *
      */
-    private void wesLaunch(final String entry, final String localEntry, final String jsonRun, final String yamlRun, final String wdlOutput, final boolean ignoreChecksumFlag, final String uuid) {
-        launchWithArgs(entry, localEntry, jsonRun, yamlRun, wdlOutput, ignoreChecksumFlag, uuid);
+    private void wesLaunch(String workflowEntry, String workflowParams, List<String> workflowAttachment) {
+        // Only supports workflows at the moment
+        WesLauncher.launchWesCommand((WorkflowClient) this, workflowEntry, workflowParams, workflowAttachment);
```

Comment:
You don't have to change it, but I don't like that downcast to WorkflowClient; it could be missed when/if we support tools. I.e., you're relying on codeflow to never hit this when it's a ToolClient; you could instead have the compiler do the check for you.

A better way might be to:

1. Have this  method always throw an error and override it WorkflowClient
1. Make this method abstract, then implement it in the subclasses, where the ToolClient implementation is to error out.

Reply:
Followed approach 2. Added an implementation to `ToolClient` and `WorkflowClient`, the `ToolClient` implementation throws an error: https://github.com/dockstore/dockstore-cli/pull/100/commits/8bf0df02649d0da6dc572bb27c3bafd0a8c2a861